build.yaml

```diff
@@ -394,15 +460,12 @@ common_cli:
     sudo cp repo/* \${DOC_ROOT}/\${REPO_PATH}
     cd \${DOC_ROOT}/\${REPO_PATH}
     # create a repo on either centos or ubuntu
-    case "\$(os)"
-    in
-      "ubuntu")
-        sudo sh -c 'dpkg-scanpackages --multiversion . | gzip > Packages.gz'
-        ;;
-      "centos")
-        sudo createrepo .
-        ;;
-    esac
+    if [ "\$(is_ubuntu)" == "true" ]; then
+      sudo sh -c 'dpkg-scanpackages --multiversion . | gzip > Packages.gz'
+    fi
+    if [ "\$(is_centos)" == "true" ]; then
+      sudo createrepo .
+    fi
```

Comment:
:exclamation: I find the new code harder to understand. The original code makes it clear that there are multiple options being chosen between. With the new code, I need to realize that `is_ubuntu` and `is_centos` are mutually exclusive and (currently!) exhaustive, and that if a new distribution is added, I should add another `if`.

(It would be even better if the original case had a default case that errored out, so one would know that another case needs to be added by a noisy failure).